docs/development/extensions-contrib/prometheus.md

```diff
@@ -90,6 +90,7 @@ Prometheus metric path is organized using the following schema:
   "dimensions" : <dimension list>, 
   "type" : <timer|counter|gauge>, 
   "conversionFactor": <conversionFactor>, 
+  "histogramBuckets": <bucket values>,
```

Comment:
```suggestion
  "histogramBuckets": <array of bucket values for timer metric>,
```

Reply:
Makes sense, pushing changes to the following commit.